app/controllers/application_controller.rb

```diff
@@ -56,8 +55,11 @@ def after_login_redirect(url_to_redirect = nil)
     redirect_to(url_to_redirect) && return unless url_to_redirect.nil?
     if current_website
       redirect_to errors_url, notice: "Logged in"
+    elsif !current_user.websites.empty?
+      set_website(current_user.websites.first)
+      redirect_to errors_url
```

Comment:
There's a situation where the 'add website wizard' pops up after logging in even if you already have a website added. It happens when the website is not set in the session. For example when logging in with github.